src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,57 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int MAX_SIZE = 10;
+    private final Pair[] elements;
```

Comment:
parametrize your pair with class type parameters
```suggestion
    private final Pair<K, V>[] elements;
```